src/main/java/com/google/api/codegen/transformer/ruby/RubySurfaceNamer.java

```diff
@@ -135,7 +138,10 @@ public String getParamTypeName(ModelTypeTable typeTable, TypeRef type) {
   }
 
   /** The type name for the message property */
-  public String getMessagePropertyTypeName(ModelTypeTable typeTable, TypeRef type) {
+  @Override
+  public String getMessagePropertyTypeName(ImportTypeTable importTypeTable, FieldType fieldType) {
+    ModelTypeTable typeTable = (ModelTypeTable) importTypeTable;
```

Comment:
Done, for this specific class.

The only other remaining site of casting from ImportTypeTable to ModelTypeTable is in CSharpSurfaceNamer, in functions that take as an argument com.google.api.tools.framework.model.Method, which should be ok because that is explicity a Gapic entity.